### PR TITLE
Create Windows installer after tests

### PR DESCRIPTION
**Pull request by damieng**

### CODE REVIEW

**Comment by jsolisu** [comment]
It´s the same idea... 

https://github.com/jsolisu/atom/commit/0816efefdad9004d887d9277592f528c618c092c

Maybe "skip-rebuild" is more clear.

**Comment by damieng** [comment]
I did consider "skip-rebuild" but that kind of implies an entire `script\build` command wouldn't do anything but it still performs a whole bunch of packaging steps.

File: script/build [hunk 1]
```diff
@@ -52,18 +53,21 @@ process.on('unhandledRejection', function (e) {
...
+  transpileCoffeeScriptPaths()
+  transpileCsonPaths()
+  transpilePegJsPaths()
+  generateModuleCache()
+  prebuildLessCache()
+  generateMetadata()
+  generateAPIDocs()
+}
+
 dumpSymbols()
```
Comment (maxbrunsfeld) [inline script/build]: Could `dumpSymbols` be moved inside the `existingArtifacts` conditional as well? Maybe something like

```js
let artifactsPromise = Promise.resolve()

if (!argv.existingArtifacts) {
  checkChromedriverVersion()
  cleanOutputDirectory()
  copyAssets()
  transpilePackagesWithCustomTranspilerPaths()
  transpileBabelPaths()
  transpileCoffeeScriptPaths()
  transpileCsonPaths()
  transpilePegJsPaths()
  generateModuleCache()
  prebuildLessCache()
  generateMetadata()
  generateAPIDocs()
  artifactsPromise = dumpSymbols()
}

artifactsPromise()
  .then(packageApplication)
  .then // ...
```

**Review by maxbrunsfeld — APPROVED** [review]
This will help a lot. Thanks @damieng!

**Pull request merged**